play-nunjucks/src/main/scala/uk/gov/hmrc/nunjucks/NunjucksRenderer.scala

```diff
@@ -208,7 +208,7 @@ class NunjucksSetup @Inject() (
     Files.copy(environment.resourceAsStream("uk/gov/hmrc/nunjucks/nunjucks-bootstrap.js").get, scriptFile.path)
 
     val libDir    = tmpDir / "lib"
-    val extractor = new WebJarExtractor(environment.classLoader)
```

Comment:
The environment.classLoader appears to only include project-specific packages and not all required packages in webjars-locator-core version 0.58. In version 0.35 of webjars-locator-core, the lookup for WebJars traversed up to the parent class loader. This behavior seems to have changed in newer versions, where it no longer does that. Therefore, simply passing environment.classLoader is not sufficient.